inc/saiswitch.h

```diff
@@ -672,22 +693,23 @@ typedef sai_status_t (*sai_connect_switch_fn)(
  *   None
  */
 typedef void (*sai_disconnect_switch_fn)(
-    void
+    _In_ sai_object_id_t switch_id
```

Comment:
This function can also be removed as we can use remove switch function. In this case, it will only disconnect from the switch, as the switch id is generated with SWITCH_ATTR_INIT_SWITCH=false.